src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -94,6 +94,7 @@
                     + "                                         ERROR,WARN,INFO,DEBUG,TRACE [default: INFO].\n"
                     + " -Q, --quickcheck-test-mode              Run in QuickCheck test mode\n"
                     + " -P <seconds>, --cm-poll-interval        Configuration manager poll interval [default: 1]\n"
+                    + " -v, --verify                            Compute and verify checksums for log entries.\n"
```

Comment:
Well, there will be cases where checksumming might not desirable (for example, logunit on ZFS.) After some more thought- and reading the code more carefully, checksums should be on a per-file basis, when checksums are off, the checksum version field will show 0x0 in the header. If a file with checksum type 0 is encountered during startup and the -no-verify flag is not set, then the logunit should refuse to start. We'll need to provide some kind of upgrade path, which will calculate the checksums and "upgrade" the logfiles... I guess this could be a clojure script...